### ISSUE TITLE
Can't use remote images if there are not 20 images in the assets folder

### ISSUE THREAD

**Issue by jackwilsdon** [banool/codenames-pictures#22]
It seems like I can't even start up the `codenames` server if there aren't 20 images in the assets folder, but this doesn't really make sense if I can specify a remote address to load images from when the server is running.

**Comment by banool** [comment]
I figure that specifying a remote image source should be optional, and any random end user should be able to stumble across the site and pick it up and play a game. Saying that I guess that's something I can establish in my own hosting of the site, I suppose others might want to host it without that constraint.

If the constraint were to be relaxed, the field that accepts the remote image source would have to be made mandatory at the least. There should also be a descriptive error message when they try to start a game without supplying such a link.

**Comment by jackwilsdon** [comment]
Yeah I was imagining it starting up normally if there's not 20 images available and making the URL field mandatory - it eases setup as I had to go and find 20 images to get it up and running.

**Comment by banool** [comment]
Yeah seems reasonable. You want to look into it? If not I can put it on the to do list.
